modules/collection/programs/bottom.nix

```
@@ -0,0 +1,47 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkOption mkEnableOption mkPackageOption;
+
+  toml = pkgs.formats.toml { };
+
+  cfg = config.rum.programs.bottom;
+in
+{
+  options.rum.programs.bottom = {
+    enable = mkEnableOption "bottom";
+
+    package = mkPackageOption pkgs "bottom" { };
+
+    settings = mkOption {
+      type = toml.type;
+      default = { };
+      example = {
+        flags = {
+          battery = true;
+          tree = true;
+        };
+        styles.battery.high_battery_color = "Pink";
+      };
+      description = ''
+        The configuration converted into TOML and written to
+        `${config.directory}/.config/bottom/bottom.toml`.
```

```suggestion
        {file}`$HOME/.config/bottom/bottom.toml`.
```

[NixOS flavored CommonMark](https://github.com/NixOS/nixpkgs/blob/master/doc/README.md#roles) here too

---

Done!